### ISSUE TITLE
Allow schema from variable name in `@row`

### ISSUE THREAD

**Issue by ericphanson** [beacon-biosignals/Legolas.jl#9]
I've been finding it helpful to add `write_table`-type functions to automatically add a project-specific schema, and therefore would like to do things like:
```julia
const MySchema = Legolas.Schema("my-schema@1")
const MyRow = Legolas.@row(MySchema, a::Int)

function write_table(path_or_io, table; kwargs...)
    Legolas.write(path_or_io, table, MySchema; kwargs...)
end
```

However, `Legolas.@row` requires a hard-coded schema name there, not the `MySchema` variable. It would be nice if this could be generalized. I'm not sure if this is even possible because I don't know think the value of `MySchema` is available at macro-expansion time. But it is a pain point so I figured I'd open an issue for discussion.

**Comment by jrevels** [comment]
Hmm I see what you mean. 

I don't think I hit this whenever I work with `Schema`s since I generally prefer "spelling" a-priori-known `Schema` values as literals instead of rebinding them to a `const`.

That being said, I'm not opposed to supporting something like this if the implementation can be sussed out

**Comment by jrevels** [comment]
I think this can be closed after #54
